Gruntfile.js

```diff
@@ -9,7 +9,7 @@ module.exports = function(grunt) {
       test: {
         options: {
           reporter: 'spec',
-          require: 'coffee-script'
+          require: 'coffee-script/register'
```

Comment:
`package.json` still has coffee-script@1.6.3 for development. So, probably need to bump that, or revert this change.